vip_spec.xsd

```diff
@@ -337,8 +340,11 @@ reference elements for polling location
         <xs:element name="Person">
           <xs:complexType>
             <xs:sequence>
+              <!-- SURVEY: Types xs:string and InternationalizedText seem
+                   to be used inconsistently here. Suggestions?
```

Comment:
Just to make my suggestion concrete, I think all fields in the Person object should be xs:string, but then "BallotProfession" should be added as InternationalizedText to the Candidate object.  I'm making this suggestion under the assumption that "BallotName" in the Candidate object means the name of the candidate as it should appear on the ballot.